src/core/getters/scalar.ts

```diff
@@ -87,6 +87,10 @@ export const getScalar = async ({
         value = 'Blob';
       }
 
+      if (item.format === 'date' || item.format === 'date-time') {
```

Comment:
Hey, thanks for the pull request. It's not only that. Axios should also parse the string into a js Date then

Reply:
It should also be an option in the config to activate because it could be a breaking change for some codebase

Reply:
I will work on that